Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/InputActionsEditorState.cs

```diff
@@ -8,13 +8,66 @@
 namespace UnityEngine.InputSystem.Editor
 {
     [System.Serializable]
+
+    internal class CutElement
+    {
+        private string id;
+        internal Type type;
+
+        public CutElement(string id, Type type)
+        {
+            this.id = id;
+            this.type = type;
+        }
+
+        public int GetIndexOfProperty(InputActionsEditorState state)
+        {
+            if (type == typeof(InputActionMap))
+            {
+                var actionMap = state.serializedObject
```

Comment:
Can't this be replaced with InputActionSerializationHelpers.GetIndex or construct an equivalent GetElement?

Reply:
fixed in [96c3270](https://github.com/Unity-Technologies/InputSystem/pull/1858/commits/96c32700fc4229f946838faf1eb88cdbacf631dc)